core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -926,6 +926,15 @@ private SqlLibraryOperators() {
         OperandTypes.STRING_STRING,
         SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction PARSE_TIME =
+      new SqlFunction(
+          "PARSE_TIME",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.DATE_NULLABLE, null,
```

Comment:
Please change the ReturnTypes to TIME_NULLABLE

Reply:
DONE.